rtoml/__init__.py

```diff
@@ -18,7 +18,7 @@ def load(toml: Union[str, Path, TextIO]) -> Dict[str, Any]:
     `Path` or file object from `open()`.
     """
     if isinstance(toml, Path):
-        toml = toml.read_text()
+        toml = toml.read_text(encoding='UTF-8')
```

Comment:
Do you think we should make this an argument of the function with default `'utf8'`?

Same for below?

As far as I remember, I've never used another encoding, but I know people sometimes need to.

Reply:
Logic is fine with me if you agree.